contrib/custom-node-e2e/create-ignition-config.sh

```diff
@@ -0,0 +1,238 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+usage() {
+    printf "Usage: %s -d CRIO_DIR -i IGNITION_OUT_DIR -b GCS_BUCKET_NAME [ -s GCS_SA_PATH ]  [ -e ]  [ -h ]\n\n" "$(basename "$0")"
+    echo "Possible arguments:"
+
+    printf "  -d\tPath to cri-o source\n"
+    printf "  -i\tPath to output directory for generated ignition config\n"
+    printf "  -b\tName of the GCS bucket for uploading bundle artifacts\n"
+    printf "  -s\tPath to GCP service account file (defaults to '' when existing gcloud credentials are used)\n"
+    printf "  -e\tWhether to turn on Evented PLEG or not\n"
+    printf "  -h\tShow this help message\n"
+}
+
+CRIO_DIR=
+IGNITION_OUT_DIR=
+GCS_BUCKET_NAME=
+GCS_SA_PATH=
+
+# used for extra config, disabled by default
+# evented pleg - use 0/1 as it's value depending upon whether crio binary should be started with --evented_pleg flag or not
+EVENTED_PLEG=0
```

Comment:
nit: I find it odd we have this as a one-off option. We could add a `CRIO_OPTIONS` variable and pass that instead?

Reply:
or we could add `CRIO_DROP_IN_FILE` and inject that into the get script below

Reply:
I missed your comment IMHO, but I added an implementation that is very similar and can allow users to chain add multiple custom crio config files.  Hope that could do the job too :)

+ removed that EVENTED_PLEG parts in process.